plugins/woocommerce-blocks/tests/e2e/bin/themes/storefront-child__block-notices-filter/functions.php

```diff
@@ -0,0 +1,3 @@
+<?php
+
+add_filter( 'woocommerce_use_block_notices_in_classic_theme', '__return_true' );
```

Comment:
This filter is used to verify that classic theme developers can trigger the block notices via filter.